storage/engine/db.cc

```diff
@@ -739,6 +756,342 @@ class DBLogger : public rocksdb::Logger {
   const bool enabled_;
 };
 
+struct Base {
```

Comment:
Add a comment as to exactly what Base is. That it provides a common interface to accessing data from both an iterator and a batch.